config/initializers/cors.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
+    # in production change origins to heroku site
```

Comment:
Saw this online while researching CORS...for development `*` is fine but for production we will want to update this. I think the heroku site we will list here is front end but not 100% sure.